.travis.yml

```diff
@@ -9,8 +9,6 @@ dist: bionic
 
 php:
   - 7.1
-  - 7.2
-  - 7.3
```

Comment:
Please restore

Reply:
7.2 and 7.3 is already extensively tested with real DBs here (half of the 44 jobs!).
Those two here have no meaning and relevance.
They are not even running on the real DBs, they only provide a smoke test on basic package compatibility. Thus only min/max needed.
Instead we should focus on getting my CHECKS job in, with phpstan and cs, as that one will provide way more useful insights on top of the matrix already existing.

We compromised on keeping your jobs for 7.2/7.3 in the matrix already, even with them also being already highly questionable in terms of overall usefulness. The likelyhood of any issue is way below << 0.001 % and the job running time/costs outweigh it already immensely.

Reply:
I've explained this many times: I have bugs that can't be found when you do this.

>half of the 44 jobs

I don't care about the number of jobs that run in travis. Having computers do lots of meticulous-pain-staking-work for us is the entire point of having CI.

> Those two here have no meaning and relevance.

Here's the real stuff I'm working on:
![versions](https://user-images.githubusercontent.com/379322/86217424-be874c00-bb7f-11ea-91f5-00c26308b130.png)

Again: If you do this, nobody can find and fix these bugs, and we can't keep propel contributors from breaking versions we support.

I don't care if PHP "tends to" do some specific behaviour or the maintainers of symfony "tend to" not do something. If you do the work and run the tests on something other than min+max, they fail.

> They are not even running on the real DBs,

Doch. It runs on a real database. All tests run against real databases.

- Agnostic is a suite that executes on sqlite without having a specific adapter in the way.
- Sqlite is literally running on sqlite
- mysql is literally running on sqlite

There's landmines between versions of symfony, php, and databases for everything. I've had to make a grid just to map this out in my other work as I try to make tests pass everywhere because there's no sane way to keep track of all these cases.

This is just my progress as of today. Half of those checkmarks were :x: on Monday.
![Screenshot from 2020-07-01 09-55-40](https://user-images.githubusercontent.com/379322/86218449-14102880-bb81-11ea-9bda-6bf2e4f355fe.png)

The reality is we have lots of external dependencies that change over time, and lots of PHP versions that are blocking some of those changes, and lots of databases to support.

If you remove coverage or visibility to the thing I've been struggling with for weeks I will not support it.

Reply:
Can you prove to me you cannot see them with the remaining ones still in place?
I only removed non relevant ones.